apps/console/src/features/applications/pages/applications.tsx

```diff
@@ -422,8 +424,12 @@ const ApplicationsPage: FunctionComponent<ApplicationsPageInterface> = (
      * @returns My Account link.
      */
     const renderTenantedMyAccountLink = (): ReactElement => {
-        if (!applicationConfig.advancedConfigurations.showMyAccount ||
-            UIConfig?.legacyMode?.applicationListSystemApps) {
+        if (
+            !applicationConfig.advancedConfigurations.showMyAccount
+            || UIConfig?.legacyMode?.applicationListSystemApps
+            // Disable the myaccount navigation box for suborgs in legacyauthz runtime.
+            || (isLegacyAuthzRuntime() && isSubOrganization())
```

Comment:
If it better to write the comment before the if clause rather than in between the condition.

Ex:
```suggestion
		/**
		 * Disable the myaccount navigation box if
		 * disabled by configs
		 * or it's a system app
		 * or sub org and legacy runtime
		 */
        if (
            !applicationConfig.advancedConfigurations.showMyAccount
            || UIConfig?.legacyMode?.applicationListSystemApps
            || (isLegacyAuthzRuntime() && isSubOrganization())
```